dali/test/python/test_backend_impl.py

```diff
@@ -21,21 +21,28 @@ def test_create_tensor():
     tensor = TensorCPU(arr, types.NHWC)
     assert_array_equal(arr, np.array(tensor))
 
-def test_create_tensor_list():
+def test_create_tensorlist():
     arr = np.random.rand(3, 5, 6)
-    tensor_list = TensorListCPU(arr, types.NHWC)
-    assert_array_equal(arr, tensor_list.as_array())
+    tensorlist = TensorListCPU(arr, types.NHWC)
```

Comment:
`types.NHWC` and all other layouts are deprecated. Use string (without leading N) instead.

Reply:
This `types.NHWC` change could be a separate PR. Having this changed only here it not  must in my opinion.

Reply:
Will do, as I will change `tensor()` to `__getitem__()`

Reply:
I think you can do something like https://github.com/pybind/pybind11/blob/master/tests/test_sequences_and_iterators.cpp#L83 and implement it in .cc file.